doc/release_notes/release_notes_3.0.rst

```diff
@@ -59,8 +85,47 @@ steps to deploy and run the tool.
 Update Configuration Options
 ============================
 
-Complete overhaul of configurator in v3.0...
-
+In v3.0, data in a launch XML are merged into the scenario XML for the new configurator.
+The highly recommended way to use upgrader tool to upgrade a pair of scenario XML and
+launch XML files into a merged file. Then use the ACRN Configurator to open the upgraded
```

Comment:
Do we have instructions somewhere on how to merge these files?

Reply:
updated.